src/reactor/mod.rs

```diff
@@ -148,19 +148,18 @@ impl Reactor {
     /// extra information with it and can be safely discarded. In the future
     /// this return value may contain information about what happened while this
     /// reactor blocked.
```

Comment:
Could the documentation here be expanded to include why it can return an error?

Reply:
Done